packages/lumx-icons/rollup.config.js

```diff
@@ -44,8 +45,32 @@ export default {
         }),
         /** Copy additional files to dist. */
         copy({
-            targets: glob.sync('*(package.json|*.js|*.d.ts|*.scss|*.md)')
-                .map(file => ({ src: path.join(__dirname, file), dest: DIST_PATH })),
+            targets: glob.sync('*(package.json|*.scss|*.md)').concat(glob.sync('override/generated/{*.scss,fonts}'))
+                .map(file => ({ src: path.join(__dirname, file), dest: path.join(DIST_PATH, path.dirname(file)) })),
         }),
     ],
 };
+
+
+// Bundle TS types in D.TS files
+const bundleType = {
```

Comment:
Is there a specific reason why we need types on the released version for our icons?

Reply:
Yes the mdi icon types are really trivial but when you import non-typed module in TS project it can produce errors or warnings (depending on the project config). We did had types before from mdi (and from the v4-to-v5 alias file), I just replaced manual type definition with automatic `.d.ts` bundling (we already use on `@lumx/react`)